README.md

```diff
@@ -29,7 +29,7 @@ Options:
   -d, --dependency <dependency...> specify dependencies to be loaded using format dependencyId@version (FHIR R4 included by default)
   -v, --version                    print goFSH version
   -h, --help                       display help for command
-  -s, --style                      specify how the output is organized into files: group-by-fsh-type (default), group-by-profile, single-file, file-per-definition
+  -s, --style                      specify how the output is organized into files: group-by-fsh-type (default), group-by-profile, single-file, file-per-definition, files-organized-by-type
```

Comment:
I agree with @cmoesel, I think it makes sense to just make this `file-per-definition`.